src/frontend/src/flows/verifiableCredentials/allowCredentials.ts

```diff
@@ -8,12 +8,33 @@ import { html, TemplateResult } from "lit-html";
 import { asyncReplace } from "lit-html/directives/async-replace.js";
 import { unsafeHTML } from "lit-html/directives/unsafe-html.js";
 
+import { getDapps, KnownDapp } from "../dappsExplorer/dapps";
+import { dappTemplate } from "../dappsExplorer/index";
+
 import DOMPurify from "dompurify";
 
 import copyJson from "./allowCredentials.json";
 
 /* A screen prompting the user to allow (or cancel) issuing verified
  * credentials */
+const getOrigin = (
```

Comment:
The comment above should be move to the template below, no?

Reply:
yes of course! Let me fix that